commit d2e57c4b
Author: Mike Dalessio <mike@37signals.com>
Date:   2025-04-05 16:37:23 -0400

    patches/libxml2/0002-Update-entities-to-remove-handling-of-ssi.patch

diff --git a/entities.c b/entities.c
index c30814d0..186534c5 100644
--- a/entities.c
+++ b/entities.c
@@ -671,30 +671,22 @@ xmlEscapeText(const xmlChar *text, int flags) {
         }
 
         if (c == 0) {
             chunkSize = 0;
             repl = BAD_CAST "";
             replSize = 0;
         } else if (c == '<') {
 	    /*
 	     * Special handling of server side include in HTML attributes
 	     */
-	    if ((flags & XML_ESCAPE_HTML) && (flags & XML_ESCAPE_ATTR) &&
-	        (cur[1] == '!') && (cur[2] == '-') && (cur[3] == '-') &&
-	        ((end = xmlStrstr(cur, BAD_CAST "-->")) != NULL)) {
-                chunkSize = (end - cur) + 3;
-                repl = cur;
-                replSize = chunkSize;
-	    } else {
                 repl = BAD_CAST "&lt;";
                 replSize = 4;
-            }
 	} else if (c == '>') {
             repl = BAD_CAST "&gt;";
             replSize = 4;
 	} else if (c == '&') {
 	    /*
 	     * Special handling of &{...} construct from HTML 4, see
 	     * http://www.w3.org/TR/html401/appendix/notes.html#h-B.7.1
 	     */
                 repl = BAD_CAST "&amp;";
                 replSize = 5;
